src/main/java/home_works/home_work5/task3/Main.java

```diff
@@ -0,0 +1,14 @@
+package home_works.home_work5.task3;
+
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<Integer> list = new ArrayList<>();
```

Comment:
the list is a bad name for var and if you don't use some specific functionality from the ArrayList class then it is better to go with the List interface